scripts/extensions/auto-tagging-widget/src/auto-tagging.tsx

```diff
@@ -61,6 +61,7 @@ export function getAutoTaggingComponent(superdesk: ISuperdesk, label: string) {
     return class AutoTagging extends React.PureComponent<IProps, IState> {
         private isDirty: (a: IAutoTaggingResponse, b: Partial<IAutoTaggingResponse>) => boolean;
         private _mounted: boolean;
+        private iMatricsFields = superdesk.instance.config.iMatricsFields ?? [];
```

Comment:
`iMatricsFields` is an object in `superdesk.config.js`, the default should be an object too instead of an array.

Reply:
it would be probably simpler if it is a list in the config so what you see there is what you get in the authoring,
rather than having to check those order values